pvac_seq/binding_filter.py

```diff
@@ -0,0 +1,201 @@
+import argparse
+import sys
+import re
+import os
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--input', type=argparse.FileType('r'),
+                        help="Input list of variants",
+                        required=True)
+    parser.add_argument('-f', '--fof', type=argparse.FileType('r'),
+                        help="FOF containing list of parsed epitope files " +
+                        "for different allele-length combinations (same sample)",
+                        required=True)
+    parser.add_argument('-o', '--output', type=argparse.FileType('w'),
+                        help="Output .xls file containing list of filtered " +
+                        "epitopes based on binding affinity for each " +
+                        "allele-length combination per gene",
+                        required=True)
+    parser.add_argument('-c', '--fold-change', type=int,
+                        help="Minimum fold change between mutant binding " +
+                        "score and wild-type score. The default is 0, which " +
+                        "filters no results, but 1 is often a sensible " +
+                        "default (requiring that binding is better to the MT " +
+                        "than WT)",
+                        default=0,
+                        required=False,
+                        dest="minimum_fold_change")
+    parser.add_argument('-b', '--binding-threshold', type=int,
+                        help="Report only epitopes where the mutant allele " +
+                        "has ic50 binding scores below this value; default 500",
+                        default=500,
+                        required=False,
+                        dest="binding_threshold")
+
+    args = parser.parse_args()
+
+    #precompile regex patterns used later
+    chromosome_name = re.compile(r'^chromosome_name')
+    netmhc_subber = re.compile(r"_netmhc")
+
+    #open the variants file and parse into variants dictionary
+    intake = args.input.readline().rstrip()
+    variants = {}
+
+    while intake != '':
+        if chromosome_name.match(intake):
+            variants['header'] = intake
+            intake = args.input.readline().rstrip()
+            continue
+
+        data = intake.split('\t')
```

Comment:
I would open the input file using csv.reader instead. That way you don't need to split things manually.